index.html

```diff
@@ -559,53 +444,59 @@ <h3>
         <h3>
           <dfn>request()</dfn> method
```

Comment:
`requestPermission()`'s section name is "requestPermission() _static_ method". Shouldn't you add "static" here too?

Reply:
yes and done